launch/testnet/seeder.go

```diff
@@ -1,15 +1,12 @@
 package main
 
 import (
-	"bytes"
-	"crypto/sha256"
-	"encoding/hex"
 	"net"
 	"os"
-	"strings"
 	"time"
 
 	cfg "github.com/dusk-network/dusk-blockchain/pkg/config"
+	cli "github.com/dusk-network/dusk-blockchain/pkg/voucher/client/utils"
```

Comment:
Nitpick: as we already have a package called `cli` (for the user console), maybe it would be better to import this as `client`?